app/helpers/components/phone_number_input_helper.rb

```diff
@@ -0,0 +1,5 @@
+module Components::PhoneNumberInputHelper
+  def render_phone_number_input(:name, :value = "")
```

Comment:
esto no es ruby válido :P
`:algo` es un simbolo, usarlo en un def para declarar un parámetro es un error de sintaxis. debería ser:

si queres usar parametros posicionales
```suggestion
  def render_phone_number_input(name, value = "")
```
o, si quisieras usar keyword arguments:
```suggestion
  def render_phone_number_input(name:, value: "")
```

`:algo = otra_cosa` estaría queriendo asignar un valor a un simbolo, que ya es un valor y es inmutable. también es un error de sintaxis.